manifest_template.yml

```diff
@@ -48,12 +48,12 @@ apigee:
               {{ NAME }}:
                 quota:
                   enabled: true
-                  limit: 300
+                  limit: 6000
                   interval: 1
                   timeunit: minute
                 spikeArrest:
                   enabled: true
-                  ratelimit: 600pm # 10 requests per second
+                  ratelimit: 100ps # 100 requests per second
```

Comment:
We recommend that if you really want 100 requests per second then the value you should set here is `6000pm`

Reply:
Thanks Ben! Changed to 6000pm and pushed. This limit is from our NFRs so we really do want it.